mk/aosp_optee.mk

```diff
@@ -86,15 +86,24 @@ $(LOCAL_PREBUILT_MODULE_FILE): $(TA_TMP_FILE)
 	@mkdir -p $(dir $@)
 	cp -uvf $< $@
 
+TA_TMP_FILE_DEPS :=
+ifneq ($(local_dependency_ta), )
+$(foreach dep,$(local_dependency_ta), $(eval TA_TMP_FILE_DEPS += $(TARGET_OUT)/lib/optee_armtz/$(dep)))
+endif
+$(TA_TMP_FILE): $(TA_TMP_FILE_DEPS)
 $(TA_TMP_FILE): PRIVATE_TA_SRC_DIR := $(LOCAL_PATH)
 $(TA_TMP_FILE): PRIVATE_TA_TMP_FILE := $(TA_TMP_FILE)
 $(TA_TMP_FILE): PRIVATE_TA_TMP_DIR := $(TA_TMP_DIR)
 $(TA_TMP_FILE): BUILD_OPTEE_OS
 	@echo "Start building TA for $(PRIVATE_TA_SRC_DIR) $(PRIVATE_TA_TMP_FILE)..."
+	@echo "O=$(TOP_ROOT_ABS)/$(OPTEE_TA_OUT_DIR)/$(PRIVATE_TA_TMP_DIR)"
+	@echo "TA_DEV_KIT_DIR=$(TOP_ROOT_ABS)/$(TA_DEV_KIT_DIR)"
+	@echo "CROSS_COMPILE_LINE=$(CROSS_COMPILE_LINE)"
```

Comment:
Certainly out of scope for this PR, but this would not be needed if `@$(cmd-echo-silent)` and `$(q)` were used instead (with the benefit that the three usual verbosity levels would be available: `make`, `make V=1` and `make -s`).

Reply:
Ok, will remove this for now. Remove as fixup patch for now. Will remove patch entirely when adding tag(s).